lib/core/src/display/shape/text/text_field/cursor.rs

```diff
@@ -313,6 +391,52 @@ impl Cursors {
         self.cursors.last().unwrap()
     }
 
+    fn block_selection_location
+    ( &mut self
+    , content       : &mut TextFieldContent
+    , from_location : TextLocation
+    , to_location   : TextLocation) {
+        self.remove_additional_cursors();
+        let cursor = self.active_cursor_mut();
+
+        let from_column = from_location.column;
+        let to_column   = to_location.column;
+        let from_line   = from_location.line;
+        cursor.select_line_range(&content.line(from_line),from_column..to_column);
+
+        let range = if from_location.line > to_location.line {
+            let from_line = from_location.line - 1;
+            to_location.line..=from_line
+        } else {
+            from_location.line+1..=to_location.line
+        };
+        for line in range {
+            self.add_cursor(TextLocation::at_document_begin());
+            let cursor = self.active_cursor_mut();
+            cursor.select_line_range(&content.line(line),from_location.column..to_location.column);
+        }
+    }
+
+    /// Selects a block from active cursor position to the nearest location from given point of
+    /// the screen.
+    pub fn block_selection(&mut self, content:&mut TextFieldContent, point:Vector2<f32>) {
+        let from_location = self.cursors.first().expect("Couldn't get first").selected_to;
+        let from_line     = from_location.line;
+        let to_line       = content.line_location_at_point(point);
+        let mut start     = from_line;
+        let mut end       = to_line;
+        if start > end {
+            std::mem::swap(&mut start, &mut end)
+        }
+        let range     = start..=end;
+        let locations = range.map(|line| {
+            content.column_location_at_point(line,point)
+        });
+        let to_column   = locations.max().expect("Couldn't get max");
```

Comment:
If I get error in console "Couldn't get max", I would never understand what it's about. Plus, there is a mising dot on the end of the error sentence. The same applies to error above "Couldn't get first".